### DISCUSSION TITLE
ux/twig-component location

### DISCUSSION THREAD

**Discussion by mmarton** [symfony/symfony#48858] [Q&A]
Hi!

Is there a speciific reason why maker-bundle makes ux/twig-components under the Twig/Component namespace and subfolder? All the docs and symfonycast uses the just a top level Component folder. I know it doesn't affect much, it just seems odd, as a component is not as closely related to twig as a twig extension. (and a bit annoying that I can't follow the tutorial or use the maker)

https://symfony.com/bundles/ux-twig-component/current/index.html
https://github.com/symfony/maker-bundle/pull/1019

Thanks

**Comment by WebMamba** [comment]
I don't see any good reasons other than separate things. Maybe you can open a PR, or an issue and let's see what people think about 😁
